Source/Extensions/URLRequest.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+public extension URLRequest {
+  var jsonPayload: Any {
+    get {
+      return httpBody.flatMap { try? JSONSerialization.jsonObject(with: $0) } ?? [:]
+    }
+    set {
+      httpBody = try? JSONSerialization.data(withJSONObject: newValue)
+    }
+  }
+
+  @available(macOSApplicationExtension 10.10, *)
```

Comment:
This only introduces a minimum version restriction for macOS application extensions. The `*` means that all other platforms can use it.